packages/storybook/src/stories/ino-checkbox/ino-checkbox.stories.ts

```diff
@@ -33,59 +34,51 @@ export default {
       return story();
     },
   ],
-} as Meta;
+  // will be used as default props for all stories
+  args: {
+    checked: false,
+    disabled: false,
+    indeterminate: false,
+    name: '',
+    selection: false,
+    value: '',
+  },
+} as Meta<Components.InoCheckbox>;
 
-export const Playground: Story<Components.InoCheckbox> = (args) => html`
-  <ino-checkbox
-    class="customizable-checkbox"
-    checked="${args.checked}"
-    disabled="${args.disabled}"
-    indeterminate="${args.indeterminate}"
-    name="${args.name}"
-    selection="${args.selection}"
-    value="${args.value}"
-  >
-    Label
-  </ino-checkbox>
-`;
+// the basic template for the checkbox component
+const template = new TemplateGenerator<Components.InoCheckbox>(
+  'ino-checkbox',
+  (args) => html`
+    <ino-checkbox
+      checked="${args.checked}"
+      disabled="${args.disabled}"
+      indeterminate="${args.indeterminate}"
+      name="${args.name}"
+      selection="${args.selection}"
+      value="${args.value}"
+    >
+      Label
+    </ino-checkbox>
+  `
+);
 
-Playground.args = {
-  checked: false,
-  disabled: false,
-  indeterminate: false,
-  name: '',
-  selection: false,
-  value: '',
-};
+export const Playground = template.generateStory({});
```

Comment:
Just on idea to make things clearer here: Rename this to `template.generatePlaygroundStory()` (default should be without passing an object)